app/models/tag.rb

```diff
@@ -130,7 +132,7 @@ def list_items_from_contentapi
 
   # FIXME: remove this once we're using content_id's in URLs everywhere.
   def panopticon_slug
-    self.base_path.gsub('/browse', '')[1..-1]
+    self.base_path.gsub('/browse', '').gsub('/topic', '')[1..-1]
```

Comment:
These `sub`'s need anchoring.  I also wonder whether this is worth moving to the 2 subclasses to make it clearer (avoid having to do both `sub`s)